### ISSUE TITLE
What should I do to train model with 1080p images

### ISSUE THREAD

**Issue by Rainweic** [Adamdad/keras-YOLOv3-mobilenet#50]
If I want to train model with 1080p image, or other size image, what should I do? The val_loss > 4000 。。。

`
Train on 5022 samples, val on 557 samples, with batch size 32.
Epoch 1/50
156/156 [==============================] - 481s 3s/step - loss: 1247.0383 - val_loss: 6076.1475
Epoch 2/50
156/156 [==============================] - 451s 3s/step - loss: 232.9773 - val_loss: 6009.3626
Epoch 3/50
156/156 [==============================] - 451s 3s/step - loss: 166.7342 - val_loss: 6068.7722
`
